build/IPDK_Container/Ubuntu20.04/scripts/build_p4sde.sh

```diff
@@ -0,0 +1,97 @@
+#Copyright (C) 2021 Intel Corporation
+#SPDX-License-Identifier: Apache-2.0
+
+#!/usr/bin/bash
+
+set -e
+
+source os_ver_details.sh
+
+if [ -z "$1" ]
+then
+   echo "-Missing mandatory arguments:"
+   echo " - Usage: ./build-p4sde.sh <WORKDIR> "
+   return 1
+fi
+
+# npm using https for git
+git config --global url."https://github.com/".insteadOf git@github.com:
+git config --global url."https://".insteadOf git://
+
+WORKDIR=$1
+cd ${WORKDIR}
+
+echo "Removing p4-sde directory if it already exists"
+if [ -d "p4-sde" ]; then rm -Rf p4-sde; fi
+mkdir $1/p4-sde && cd $1/p4-sde
+#..Setting Environment Variables..#
+echo "Exporting Environment Variables....."
+export SDE=${PWD}
+export SDE_INSTALL=$SDE/install
+
+#...Package Config Path...#
+export PKG_CONFIG_PATH=${SDE_INSTALL}/lib64/pkgconfig
```

Comment:
Once again please make sure PKG_CONFIG_PATH is configured correctly.
By default, DPDK .pc and installation files are placed with different folder names.
If I remember correctly, Ubuntu 18.04 default installation path folder names were different when we compared with FC 33.

Reply:
Fixed it.